README.md

```diff
@@ -1 +1,52 @@
-# WebPack
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Awsome Books
+
+> In this project, we design a web page that creates a list of books. You can add a new book and remove a book from the list. The list is saved dynamically on the local storage, so every time you reload the page, the data saved on the local storage is displayed on the page.
+
```

Comment:
- [OPTIONAL] This is a **To-Do list** project. In that case, I think you need to change instead to **Awesome Books**
Please try to make this change and also look at the description of the project as well.